zooniverse/front-end-monorepo
# lib-user: Add Layout

## Package
- lib-user
- app-root

## Describe your changes
- adds Layout component per [related Figma designs](https://www.figma.com/file/qbqbmR3t5XV6eKcpuRj7mG/Group-Stats?type=design&node-id=457-5858&mode=dev) to lib-user
- adds @zooniverse/user dependency to app-root
- adds UserStats component to app-root/users/[login]/stats page.js

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What user actions should my reviewer step through to review this PR?
  - run `yarn dev` in app-root
  - go to https://local.zooniverse.org:3000/users/[login]/stats
  - adjust browser window width, note how different widths compare to linked Figma designs
- Which storybook stories should be reviewed? n/a
- Are there plans for follow up PR’s to further fix this bug or develop this feature? yes, see linked GitHub project

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## New Feature
  - Example of SlideTutorial [component](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js) with docgen comments, and its [story](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/docs/other-slidetutorial--default)
